cmd/ocsp-responder/main.go

```diff
@@ -92,11 +96,11 @@ type Config struct {
 		// instance, if the timeout is 5 seconds, and a signing takes 20ms,
 		// and we have MaxInflightSignings = 40, we can expect to process
 		// 40 * 5 / 0.02 = 10,000 requests before the oldest request times out.
-		MaxSigningWaiters int
+		MaxSigningWaiters int `validate:"min=0"`
 
-		ShutdownStopTimeout config.Duration
+		ShutdownStopTimeout config.Duration `validate:"-"`
```

Comment:
This doesn't appear to have a default value, and passing the zero value of a time.Duration one line 248 is going to be unfortunate. Why is this marked as unvalidated?